spec/maths/statistics/vector_spec.rb

```diff
@@ -710,7 +710,7 @@
   end
 
   RSpec.shared_examples 'correct macd' do |*args|
-    let(:source) { Daru::DataFrame.from_csv('spec/fixtures/macd_data.csv') }
+    let(:source) { Daru::DataFrame.read_csv('spec/fixtures/macd_data.csv') }
```

Comment:
I somehow want to keep out the IO methods from the tests of daru. For example, a simple parsing error with daru-io could be misunderstood as a logical error with the method (of course, unless the error log is seen). Only 2 more fixture files are remaining - `macd_data.csv` and `duplicates.csv`. Can a truncated version of both these files be fed as manual data in the specs and then maybe remove both the files?

Reply:
Well, for me an ability to use CSV fixtures in general Daru tests seems important, especially for future (more complicated stats methods and stuff). So let's leave this as is.

And I believe that daru-io errors would be clearly seen in specs, if the problem is daru-io